example/ck_tile/36_pooling/pool3d.cpp

```diff
@@ -171,8 +181,15 @@ bool run(const ck_tile::ArgParser& arg_parser)
             in, out_ref, kernel_args, ReduceOp{});
         out_buf.FromDevice(out.mData.data());
         pass = ck_tile::check_err(out, out_ref);
+        std::cout << "value_valid:" << (pass ? "y" : "n") << std::flush << std::endl;
```

Comment:
Flush is unnecessary. `std::endl` is already doing that. 
```suggestion
        std::cout << "value_valid:" << (pass ? "y" : "n") << std::endl;
```